corehq/apps/userreports/models.py

```diff
@@ -467,6 +473,20 @@ def _filter_interploation_helper(self, config_filter, property_name):
             return prop_value
         return [None]
 
+    @property
+    def pk_columns(self):
+        if not self.sql_settings.primary_key:
+            columns = []
+            for col in self.get_columns():
+                if col.is_primary_key:
+                    column_name = col.database_column_name
+                    if isinstance(column_name, bytes):
+                        column_name = column_name.decode('utf-8')
```

Comment:
out of curiosity what use case is this handling?

Reply:
It is meant to mirror https://github.com/dimagi/commcare-hq/blob/dae2862984d1f728ced1031ca60463bf66ee4606/corehq/apps/userreports/sql/columns.py#L14-L16 so that the column name, and name I pass into the `PrimaryKeyConstraint` are the same.

Reply:
gotcha. if you wanted you could make a tiny helper function just to make that more explicit.